test/files/pos/t6734.scala

```diff
@@ -0,0 +1,23 @@
+
+//single file badimp.scala
+// adding package object gives not found: type SortedMap
+package object badimp
+
+package badimp {
+
+  // move before package object works
```

Comment:
This comment is more confusing than clarifying. This test case is the way it is now, and the ticket you link to gives the background and other explanation.